### PR TITLE
fix typos in flash_shell

### PR DESCRIPTION
**Pull request by dale-herman**

### CODE REVIEW

**Comment by henrikbrixandersen** [comment]
@dale-herman Your PR contains a commit to the ISO-TP protocol as well as changes to the flash shell. Yet, without a proper description of the your PR (it only has a title), it is difficult to know what your intentions are.

Please fix this by limiting the PR to one isolated change and provide a proper description for the reviewers.

**Review by henrikbrixandersen — CHANGES_REQUESTED** [review]
Please limit the PR to one functional change.

**Comment by henrikbrixandersen** [comment]
@dale-herman ping

File: subsys/canbus/isotp/isotp_internal.h [hunk 1]
```diff
@@ -27,11 +27,11 @@
  * ST      Separation time
  * PCI     Process Control Information
  */
-
+#define ISOTP_USE_CAN_FD 1
```
Comment (FRASTM) [inline subsys/canbus/isotp/isotp_internal.h:30]: defining the ISOTP_USE_CAN_FD, then ISOTP_USE_CAN_FD is defined : line 32 is true

File: drivers/adc/adc_stm32.c [hunk 2]
```diff
@@ -1595,7 +1595,7 @@ static const struct adc_driver_api api_stm32_driver_api = {
 
 bool adc_stm32_is_irq_active(ADC_TypeDef *adc)
 {
-#if defined(CONFIG_SOC_SERIES_STM32G4X)
+#if defined(CONFIG_SOC_SERIES_STM32G4X) || defined(CONFIG_SOC_SERIES_STM32H7X)
```
Comment (gautierg-st) [inline drivers/adc/adc_stm32.c:1598]: Please rebase to latest main, this part is reworked and fixed.

**Pull request closed without merging**